ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_core.cc

```diff
@@ -57,24 +56,57 @@ TFCore::TFCore (const char * _model_path)
  */
 TFCore::~TFCore ()
 {
+  TF_DeleteGraph (graph);
+
+  TF_Status* status = TF_NewStatus ();
+  TF_CloseSession (session, status);
+  if (TF_GetCode (status) != TF_OK) {
+    g_critical ("Error during session close!! - [Code: %d] %s",
+      TF_GetCode (status), TF_Message (status));
+    TF_DeleteStatus (status);
+  }
+
+  TF_DeleteSession (session, status);
+  if (TF_GetCode (status) != TF_OK) {
+    g_critical ("Error during session delete!! - [Code: %d] %s",
+      TF_GetCode (status), TF_Message (status));
+    TF_DeleteStatus (status);
+  }
+  TF_DeleteStatus (status);
```

Comment:
line 66 and 73 TF_DeleteStatus()
TF_DeleteStatus() will delete status instance and this may cause an error (access to deallocated)